src/lib/template/templatizer.html

```diff
@@ -241,10 +243,12 @@
         // Instance setup
         if (template != this) {
           Polymer.Bind.prepareInstance(template);
-          template._forwardParentProp =
-            this._forwardParentProp.bind(this);
+          template._forwardParentProp = this._forwardParentProp.bind(this);
         }
         this._extendTemplate(template, proto);
+        if (template != this) {
+          template._pathEffector = this._pathEffectorTemplate.bind(this);
```

Comment:
Consider instead unconditionally doing:

`template._pathEffector = this._pathEffector.bind(template, this);`